docs/installing-nightly.md

```diff
@@ -0,0 +1,15 @@
+# Installing the "Nightly" build of bicep CLI and VS Code extension
+
+>**Note**: only install the nightly if you'd like to try the bleeding edge capabilities of bicep. These are much more likely to have undiscovered bugs or other issues.
+
+We are not currently publishing "nightly" releases, but you can grab the latest bits by viewing the latest Action workflows for the `main` branch (or any other branch).
+
+The easiest way to get these artifacts is through the GitHub site. Follow [this link](https://github.com/Azure/bicep/actions) to view the latest Action workflows. Find the most recent build on the `main` branch and select it:
+
+![](./images/bicep-select-action.png)
```

Comment:
This link doesn't work. I think because of the casing difference on the checked in file extension ".PNG" vs the link ".png"

Reply:
good catch - it was working locally. Fixed.